src/sentry/replays/endpoints/organization_replay_count.py

```diff
@@ -27,7 +26,7 @@
 
 class ReplayDataSourceValidator(serializers.Serializer):
     data_source = serializers.ChoiceField(
-        choices=((Dataset.Discover), (Dataset.IssuePlatform)), default=Dataset.Discover
+        choices=(("discover"), ("issue_platform")), default="discover"
```

Comment:
`choices=(ds.value for ds in [Dataset.Discover, Dataset.IssuePlatform],default=Dataset.Discover.value)`
or something like that ^^